.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
-	branch = bleeding
+	url = https://github.com/sanman00/SpongeAPI.git
```

Comment:
No need to change the submodule since we have to update the submodule anyways when merging.